xwiki-rendering-macros/xwiki-rendering-macro-box/src/test/resources/macrobox1.test

```diff
@@ -13,6 +13,7 @@ beginDocument
 beginMacroMarkerStandalone [box] [] [* one
 * two]
 beginGroup [[class]=[box]]
+beginMetaData
```

Comment:
I guess you just forgot to update the tests after adding the metadata value to InlineEditableContentBlock ?

Reply:
Updated in another commit but forgot to push :)